cmd/root.go

```diff
@@ -245,6 +246,28 @@ Configuration using environment variables
       CSQL_PROXY_INSTANCE_CONNECTION_NAME_1=my-other-project:us-central1:my-other-server \
           ./cloud-sql-proxy
 
+Configuration using a configuration file
+
+  Instead of using CLI flags, the Proxy may be configured using a configuration
+  file. The configuration file is a TOML, YAML or JSON file with the same keys as the
+  environment variables. The configuration file is specified with the
+  --config-file flag. An invocation of the Proxy using a configuration file
+  would look like the following:
+
+	  ./cloud-sql-proxy --config-file=config.toml
```

Comment:
Let's format this to wrap at 80 characters as much as possible, and move this example just two additional spaces in (aligning with above).

Reply:
Ditto below.

Reply:
in my editor is looks correct. But tried to replace the indentation with a copy of the above indentation